docs/creating-fakes.md

```diff
@@ -8,6 +8,17 @@ var foo = A.Fake<IFoo>();
 ```
 This will return a faked object that is an actual instance of the type specified (`IFoo` in this case).
 
+You can also create a collection of fakes by writing:
+```csharp
+var foos = A.CollectionOfFake<Foo>(10);
+```

```

Comment:
Not really related to this issue, but I noticed `CollectionOfFake<T>` wasn't introduced. It would have been weird to mention the non-generic one without mentioning the generic one, so I added it as well.